src/ServiceBusChannel.php

```diff
@@ -163,9 +145,32 @@ private function getUrl($endpoint): string
 
     public function generateTokenKey()
```

Comment:
this should be a private method

Reply:
@norbybaru There is a pre-existing test that is using this. Otherwise I would have made it private